app/src/androidTest/java/sdp/moneyrun/RiddlesDatabaseTest.java

```diff
@@ -0,0 +1,62 @@
+package sdp.moneyrun;
+
+import android.content.Context;
+
+import junit.framework.TestCase;
+import androidx.test.core.app.ApplicationProvider;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class RiddlesDatabaseTest extends TestCase {
+
+    private Context context = ApplicationProvider.getApplicationContext();
+
+
+    @Test
+    public void testGetInstance() {
+        RiddlesDatabase db = RiddlesDatabase.createInstance(context);
+        assertEquals(RiddlesDatabase.getInstance(), db);
+        RiddlesDatabase.reset();
+
+    }
+
+    @Test
+    public void testGetInstanceOnNonExistentThrowsException() {
```

Comment:
That test can never fail because JUnit tests don't return false if they encounter no assert they return true.
```suggestion
    @Test(expected = RunTimeException.class)
    public void testGetInstanceOnNonExistentThrowsException() {
       RiddlesDatabase db = RiddlesDatabase.getInstance();
       .....
       } //Not sure how the db.reset() works so I'll that to you to figure 
```

Reply:
Oh, I did not know that, thanks for pointing it out, I will fix it.